dcor/_dcor.py

```diff
@@ -1169,3 +1175,131 @@ def distance_correlation_af_inv(
             compile_mode=compile_mode,
         ),
     )
+
+
+
+
+
+"""
+A Statistically and Numerically Efficient Independence Test Based on
+     Random Projections and Distance Covariance
+
+:cite:`b-dcov_random_projection`.
+
+References
+----------
+.. bibliography:: ../refs.bib
```

Comment:
I would really appreciate it if you kindly guide me how to do that, I did search this names on the internet, but I am not quite certain how to do it. It would be really nice if you guide me step-by-step process.

Reply:
Just use `:footcite:` instead of `:cite:` inside the text to place the footnote, and `.. footbibliography::` instead of `.. biblioggraphy::`. You do not need to specify the .bib file, as it is done in the global config.

You can use this docstring as an example:
https://github.com/GAA-UAM/scikit-fda/blob/3eb8ace81220011559df83be0ffb9154a11fb001/skfda/exploratory/stats/_fisher_rao.py#L66-L99

Reply:
I implemented the exact format that you have mentioned